moduleroot/.github/workflows/acceptance.yml.erb

```diff
@@ -0,0 +1,54 @@
+<%- if Dir[File.join(@metadata[:workdir], 'spec', 'acceptance', '**', '*_spec.rb')].any? -%>
+name: Acceptance tests
+
+on:
+  - pull_request
+  - push
```

Comment:
Should we run it after pushing? Feels like after the PR and relying on the cron should be sufficient and saves a lot of resources.

Reply:
I do not think the push is needed. Only if/when we get into auto-releasing would the push be useful to verify things before the release is actually made.